adapter/src/main/resources/check_health.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
cant we provide as shell script args. In that case, we would only need one script.